misc/word_palindrome.c

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file
+ * @brief Check if a given word or phrase is a palindrome.
+ * @details
+ * A palindrome is a word, number, phrase, or other sequence of 
+ * characters which reads the same backward as forward, such as
+ * madam, racecar.
+ * This program will be able to check phrases, with words 
+ * separated by spaces, but can't deal with special characters.
+ * @author [Maria Eduarda Kawakami](https://github.com/madukm)
+ *
+ */
+
+#include <stdlib.h> //for malloc and free
+#include <assert.h> //for assert
+#include <string.h> //for strlen 
+#include <ctype.h> //for tolower
```

Comment:
```suggestion
#include <stdlib.h> /// for malloc and free
#include <assert.h> /// for assert
#include <string.h> /// for strlen 
#include <ctype.h> /// for tolower
```

Reply:
/// is standard? Should I always use it like that?

Reply:
Only for one-line variables and headers/libraries.
If you want to document some variables, do it like this:

```c++
uint32_t num1 = 0;  ///< brief documentation
```